### ISSUE TITLE
tapViewWithAccessibilityLabel: doesn't scroll to offscreen collection view cells

### ISSUE THREAD

**Issue by hiltonc** [kif-framework/KIF#376]
The problem is that if I use `tapViewWithAccessibilityLabel:` or `waitForViewWithAccessibilityLabel:` to tap or find a collection view cell that is offscreen, it fails. Comparable code works for an offscreen table view row.

I wrote a test case to illustrate the issue at https://github.com/GalacticMegacorp/KIF/commit/98824f9be5501ca96990b309c2a5936b7d2e954b (test method `testTappingItemsByLabel`). It includes a comparable test for table view that succeeds (test method `testTappingRowsByLabel`).
